ext/pgsql/pgsql.c

```diff
@@ -5433,9 +5474,16 @@ PHP_FUNCTION(pg_insert)
 							  &pgsql_link, &table, &table_len, &values, &option) == FAILURE) {
 		RETURN_THROWS();
 	}
+
+	if (table_len == 0) {
+		zend_argument_value_error(2, "cannot be empty");
+		RETURN_THROWS();
+	}
+
 	if (option & ~(PGSQL_CONV_OPTS|PGSQL_DML_NO_CONV|PGSQL_DML_EXEC|PGSQL_DML_ASYNC|PGSQL_DML_STRING|PGSQL_DML_ESCAPE)) {
-		php_error_docref(NULL, E_WARNING, "Invalid option is specified");
-		RETURN_FALSE;
+		zend_argument_value_error(4, "must be a valid bit mask of PGSQL_CONV_FORCE_NULL, PGSQL_DML_NO_CONV, "
+    		"PGSQL_DML_ESCAPE, PGSQL_DML_EXEC, PGSQL_DML_ASYNC or PGSQL_DML_STRING");
```

Comment:
```suggestion
    		"PGSQL_DML_ESCAPE, PGSQL_DML_EXEC, PGSQL_DML_ASYNC and PGSQL_DML_STRING");
```
Also you have an inconsistency here regarding oxford comma usage.

Reply:
Same note for occurrences below.